src/Stratis.Features.FederatedPeg.IntegrationTests/StratisXSyncTests.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices;
+using NBitcoin;
+using Stratis.Bitcoin.Features.RPC;
+using Stratis.Bitcoin.IntegrationTests.Common.EnvironmentMockUpHelpers;
+using Stratis.Bitcoin.IntegrationTests.Common.ReadyData;
+using Stratis.Bitcoin.Networks;
+using Stratis.Bitcoin.Tests.Common;
+using Stratis.Features.FederatedPeg.IntegrationTests.Utils;
+using Stratis.Sidechains.Networks;
+using Xunit;
+
+namespace Stratis.Features.FederatedPeg.IntegrationTests
```

Comment:
Why the test is in federated peg tests and using cirrus over the testnet?

Reply:
I don't quite follow? This is to check that the mainnet gateway nodes can connect to Stratis right?